src/Components/Blazor/Blazor/src/Services/WebAssemblyConsoleLogger.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Blazor.Services
+{
+    internal class WebAssemblyConsoleLogger<T> : ILogger<T>
```

Comment:
This is intended as a minimal way of meeting the requirements for the built-in default `IAuthorizationService`, without bringing in the full-fledged `Microsoft.AspNetCore.Extensions.Logging`.

The full logging package adds about 100KB to the app size and a couple extra assembly downloads, so if we can have a bare-bones one without it that's preferable. People who want to use the full thing (e.g., to control logging levels, have multiple categories, etc.) can still reference it and add it to their DI config.

Reply:
>The full logging package adds about 100KB to the app size and a couple extra assembly downloads, so if we can have a bare-bones one without it that's preferable. People who want to use the full thing (e.g., to control logging levels, have multiple categories, etc.) can still reference it and add it to their DI config.

To me this seems like a great thing to chase down in the future when we have a TFM for web assembly. Right now if we wanted to offer a thinner profile of these features for Blazor we have no real way to do so.